bin/antigen.zsh

```diff
@@ -560,13 +551,8 @@ antigen () {
 }
 
 -antigen-use-prezto () {
-  _zdotdir_set=${+parameters[ZDOTDIR]}
-  if (( _zdotdir_set )); then
-    _old_zdotdir=$ZDOTDIR
-  fi
-  export ZDOTDIR=$ADOTDIR/repos/
-
-  antigen-bundle $ANTIGEN_PREZTO_REPO_URL
+  export ZPREZTODIR="$(-antigen-get-clone-dir "$ANTIGEN_PREZTO_REPO_URL")"
```

Comment:
This line isn't needed, and has no effect since [ZPREZTODIR is reset regardless](https://github.com/zsh-users/prezto/commit/037c781a1600b7dc8d6cb3e10e7a75d3c79c3853#diff-70e532346dbd4347d1a49387eb6c552cR93).